lego/apps/meetings/serializers.py

```diff
@@ -150,3 +155,34 @@ class Meta:
             "start_time",
         )
         read_only = True
+
+
+class MeetingTemplateSerializer(BasisModelSerializer):
+    report = ContentSerializerField()
+    report_author = PublicUserField(
+        queryset=User.objects.all(), allow_null=True, required=False
+    )
+    invited_users = PublicUserField(
+        queryset=User.objects.all(), allow_null=True, required=False, many=True
+    )
+    invited_groups = AbakusGroupField(
+        queryset=AbakusGroup.objects.all(), allow_null=True, required=False, many=True
+    )
+    location = serializers.CharField(required=False, allow_null=True)
+
+    class Meta:
+        model = MeetingTemplate
+        fields = (
+            "id",
+            "name",
+            "report",
+            "location",
+            "start_time",
+            "end_time",
+            "description",
+            "mazemap_poi",
+            "report_author",
+            "invited_users",
+            "invited_groups",
+            "created_by",
+        )
```

Comment:
Again, this is basically copy paste from a meeting serializer, right?

Reply:
Only report and report_author are the same. I could move it out to a different basismeetingserializer class or something but I dont see that much point?